src/toolbars/MeterToolBar.cpp

```diff
@@ -234,6 +234,7 @@ void MeterToolBar::Populate()
       mRecordSetupButton = safenew AButton(this);
       mRecordSetupButton->SetLabel({});
       mRecordSetupButton->SetName(_("Record Meter"));
+      mRecordSetupButton->SetToolTip(XO("Input Metering"));
```

Comment:
Could you please change text to `"Record Meter"` and `"Playback Meter"` respectively? They already have a translation

Reply:
Done.